nixos/modules/programs/iio-hyprland.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options = {
+    programs.iio-hyprland = {
+      enable = mkOption {
+        description = lib.mdDoc ''
+          Enable this option to get iio.hyprland and get/start iio-sensor-proxy.
+        '';
+        type = types.bool;
+        default = false;
+      };
+    };
+  };
+
+  config = mkIf config.programs.iio-hyprland.enable {
+    hardware.sensor.iio.enable = lib.mkDefault true;
```

```suggestion
    hardware.sensor.iio.enable = mkDefault true;
```
Not needed since `with lib;`

---

`with lib;` should be discouraged instead

---

I have removed the `with lib;` instead and (hopefully) added `lib.` wherever it is needed instead.